### PR TITLE
Fix assert fail

### PR DESCRIPTION
**Pull request by jschneidereit**
Decided to ignore Inconclusive so the owner of the test project has to make a decision.

### CODE REVIEW

File: MsTestToXunitConverter/AssertRewriter.cs [hunk 1]
```diff
@@ -11,7 +12,7 @@ namespace MsTestToXunitConverter
     /// </summary>
     internal static class AssertRewriter
     {
-        private static InvocationExpressionSyntax InvocationRewriter(this InvocationExpressionSyntax invocation, string from, string to, IdentifierNameSyntax identifier = null)
+        private static InvocationExpressionSyntax InvocationRewriter(this InvocationExpressionSyntax invocation, string from, string to, IdentifierNameSyntax identifier = null, Func<InvocationExpressionSyntax, InvocationExpressionSyntax> func = null)
         {
             if (invocation.Expression.Kind() != SyntaxKind.SimpleMemberAccessExpression)
             {
```
Comment (RikkiGibson) [inline MsTestToXunitConverter/AssertRewriter.cs:18]: re line 22: I don't understand why "from" is an argument to this method.

**Comment by RikkiGibson** [comment]
So .Inconclusive is a build error? I guess it would be weird if people had more than few of those, even in a big test suite. I still think a test failure would be better.

**Review by RikkiGibson — APPROVED** [review]
LGTM

File: MsTestToXunitConverter/AssertRewriter.cs (hunk 1 above)
Comment (jschneidereit) [reply MsTestToXunitConverter/AssertRewriter.cs:18]: So you can go `from` something `to` another thing. e.g.: from `Assert.Fail` to `True`

**Pull request merged**